components/terrain/quadtreeworld.cpp

```diff
@@ -425,7 +426,7 @@ void QuadTreeWorld::enable(bool enabled)
     }
 
     if (mRootNode)
-        mRootNode->setNodeMask(enabled ? ~0 : 0);
+        mRootNode->setNodeMask(enabled ? ~SceneUtil::Mask_Disabled : SceneUtil::Mask_Disabled);
```

Comment:
Shouldn't ~SceneUtil::Mask_Disabled be Mask_Default instead? Since we are moving away from magic numbers anyway. And the naming is a bit weird. How about renaming Mask_Disabled to Mask_None and Mask_Default to Mask_All?

Reply:
> Shouldn't ~SceneUtil::Mask_Disabled be Mask_Default instead?

Well, if bitwise negation applied to 0 produces the 0xffffffff result on all supported platforms, it should be safe to use the Mask_Default here.

> How about renaming Mask_Disabled to Mask_None and Mask_Default to Mask_All?

I do not like the Mask_All here since in our current implementation we can not combine mask. So `Mask_All != Mask_Object|Mask_Terrain|Mask_Debug|... ` It just means that the node has the default node mask, without any special meaning.

Reply:
I think I prefer `All` over` `Default`, but maybe `Any` is a better choice.